TileStache/Goodies/VecTiles/transform.py

```diff
@@ -3185,3 +3185,41 @@ def elevation_to_meters(shape, props, fid, zoom):
 
     props['elevation'] = _to_float_meters(elevation)
     return shape, props, fid
+
+
+def normalize_cycleway(shape, props, fid, zoom):
+    """
+    If the properties contain both a cycleway:left and cycleway:right
+    with the same values, those should be removed and replaced with a
+    single cycleway property. Additionally, if a cycleway_both tag is
+    present, normalize that to the cycleway tag.
+    """
+    cycleway = props.get('cycleway')
+    cycleway_left = props.get('cycleway_left')
+    cycleway_right = props.get('cycleway_right')
+
+    cycleway_both = props.pop('cycleway_both', None)
+    if cycleway_both and not cycleway:
+        props['cycleway'] = cycleway = cycleway_both
```

Comment:
I assume this is right-associative? (`props['cycleway'] = (cycleway = cycleway_both)`)?